src/shared/Core/Constants.cs

```diff
@@ -49,6 +49,7 @@ public static class EnvironmentVariables
             public const string GcmTrace              = "GCM_TRACE";
             public const string GcmTraceSecrets       = "GCM_TRACE_SECRETS";
             public const string GcmTraceMsAuth        = "GCM_TRACE_MSAUTH";
+            public const string GcmTrace2             = "GCM_TRACE2";
```

Comment:
A big picture question: Are we always planning to have a separate `GCM_TRACE2` variable, or will we be piggy-backing `GIT_TRACE2*` variables eventually?

Reply:
Yeah, I'm wondering about whether we should read a GCM_TRACE2 or the GIT_TRACE2_EVENT and/or `git config trace2.eventtarget`.

Reply:
I think it probably does make more sense to piggyback here - my initial thinking was to give users a bit more control over whether trace2 information is sent, but it could prove to be a hassle to enable for both Git and GCM.

Reply:
+1 for using just `GIT_` vars.